AsaApi/Core/Public/API/ARK/Inventory.h

```diff
@@ -960,15 +960,15 @@ struct UPrimalInventoryComponent : UActorComponent
     bool CanInventoryItem(UPrimalItem* anItem) { return NativeCall<bool, UPrimalItem*>(this, "UPrimalInventoryComponent.CanInventoryItem(UPrimalItem*)", anItem); }
     bool CheckFullInventoryConditionForItem(UPrimalItem* anItem) { return NativeCall<bool, UPrimalItem*>(this, "UPrimalInventoryComponent.CheckFullInventoryConditionForItem(UPrimalItem*)", anItem); }
     bool AllowAddInventoryItem(UPrimalItem* anItem, int* requestedQuantity, bool OnlyAddAll) { return NativeCall<bool, UPrimalItem*, int*, bool>(this, "UPrimalInventoryComponent.AllowAddInventoryItem(UPrimalItem*,int*,bool)", anItem, requestedQuantity, OnlyAddAll); }
-    UPrimalItem* AddItem(const FItemNetInfo* theItemInfo, bool bEquipItem, bool bAddToSlot, bool bDontStack, FItemNetID* InventoryInsertAfterItemID, bool bShowHUDNotification, bool bDontRecalcSpoilingTime, bool bForceIncompleteStacking, AShooterCharacter* OwnerPlayer, bool bIgnoreAbsoluteMaxInventory, bool bInsertAtItemIDIndexInstead, bool doVersionCheck) { return NativeCall<UPrimalItem*, const FItemNetInfo*, bool, bool, bool, FItemNetID*, bool, bool, bool, AShooterCharacter*, bool, bool, bool>(this, "UPrimalInventoryComponent.AddItem(FItemNetInfo*,bool,bool,bool,FItemNetID*,bool,bool,bool,AShooterCharacter*,bool,bool,bool)", theItemInfo, bEquipItem, bAddToSlot, bDontStack, InventoryInsertAfterItemID, bShowHUDNotification, bDontRecalcSpoilingTime, bForceIncompleteStacking, OwnerPlayer, bIgnoreAbsoluteMaxInventory, bInsertAtItemIDIndexInstead, doVersionCheck); }
+    UPrimalItem* AddItem(const FItemNetInfo* theItemInfo, bool bEquipItem, bool bAddToSlot, bool bDontStack, FItemNetID* InventoryInsertAfterItemID, bool bShowHUDNotification, bool bDontRecalcSpoilingTime, bool bForceIncompleteStacking, AShooterCharacter* OwnerPlayer, bool bIgnoreAbsoluteMaxInventory, bool bInsertAtItemIDIndexInstead, bool doVersionCheck) { return NativeCall<UPrimalItem*, const FItemNetInfo*, bool, bool, bool, FItemNetID*, bool, bool, bool, AShooterCharacter*, bool, bool, bool>(this, "UPrimalInventoryComponent.AddItem(FItemNetInfo&,bool,bool,bool,FItemNetID*,bool,bool,bool,AShooterCharacter*,bool,bool,bool)", theItemInfo, bEquipItem, bAddToSlot, bDontStack, InventoryInsertAfterItemID, bShowHUDNotification, bDontRecalcSpoilingTime, bForceIncompleteStacking, OwnerPlayer, bIgnoreAbsoluteMaxInventory, bInsertAtItemIDIndexInstead, doVersionCheck); }
```

Comment:
Native calls is using references in the function names, but the function signatures are still taking pointers.

Reply:
pushed a commit to correct